source/development/packaging/generate-dashboard-package.rst

```diff
@@ -0,0 +1,186 @@
+===============
+Wazuh Dashboard
+===============
+
+The packages' generation process is orchestrated by one script, which is
+found within the ``dev-tools/build-packages/build-packages.sh`` folder of the repository:
+
+- ``build-packages.sh``: This script is responsible for bundling plugins into 1 single application in tar, rpm and/or deb distributions. With the parameters ``version``, ``revision``, ``distribution``,  package of ``wazuh-dashboard``, ``wazuh-dashboard-plugins`` and ``wazuh-security-dashboards-plugin``.
+
+Official packages are built through a GitHub Actions pipeline, however,
+the process is designed to be independent enough for maximum
+portability. The building process is self-contained in the application
+code.
+
+Build manually
+^^^^^^^^^^^^^^
+
+1. To use the scrip you first need to generate the packages from the repositories:
+
+- ``wazuh-dashboard``
+- ``wazuh-security-dashboards-plugin`` 
+- ``wazuh-dashboard-plugins``
+
+To do so, follow these steps:
+
+   1.1. Clone the Wazuh Dashboard repository and build the application.
+
+   .. code:: console
+
+      # git clone -b <branch> https://github.com/wazuh/wazuh-dashboard.git
+      # cd wazuh-dashboard/
+      # yarn osd bootstrap
+      # yarn build --linux --skip-os-packages --release
+
+   1.2. Clone the Wazuh Security Dashboards Plugin and build the plugin.
```

Comment:
It may be useful to explain that it should be cloned inside of the `plugins` folder, regardless of the command that  already does that